src/lib.rs

```diff
@@ -85,8 +85,20 @@
 //! It should work on any POSIX.1-2001 system, which are all the major big OSes with the notable
 //! exception of Windows.
 //!
-//! Windows has some limited support for signals, patches to include support in this library are
-//! welcome.
+//! This crate includes a limited support for Windows, based on `signal`/`raise` in the CRT.
+//! There are differences in both API and behavior:
+//!
+//! - `iterator` and `pipe` are not yet implemented.
+//! - We have only a few signals: `SIGABRT`, `SIGFPE`, `SIGILL`, `SIGINT`, `SIGSEGV` and `SIGTERM`.
+//!   - CRT headers also define `SIGABRT_COMPAT` and `SIGBREAK`,
+//!     but it's not exported from libc yet.
```

Comment:
:thinking: Would you know if the constants are guaranteed to be the same everywhere? We could define them ourselves in such case. Not necessarily in scope of this PR, though.

Reply:
In Windows SDK 10, they're defined in `C:\Program Files (x86)\Windows Kits\10\Include\%VERSION%\ucrt\signal.h` without `#ifdef` conditionals. So I guess they're uniform between architectures. I'm in favor of defining `SIGABRT_COMPAT`/`SIGBREAK` here and backport to `libc` if we can.